kartothek/utils/predicate_converter.py

```diff
@@ -60,7 +60,9 @@ def write_predicate_as_cube_condition(predicate: Tuple[str, str, Any]) -> Any:
     return condition
 
 
-def convert_predicates_to_cube_conditions(predicates: List[List],) -> Any:
+def convert_predicates_to_cube_conditions(
+    predicates: List[List],
```

Comment:
The interesting part here is the typing of the inner list. Reading the docstring I'm confused whether the inner but should be a list as it does contain exactly one element and thus should rather be this one element directly.